libraries/standard/mqtt/include/mqtt.h

```diff
@@ -26,6 +26,13 @@
 #include "mqtt_config.h"
 #include "mqtt_lightweight.h"
 
+/**
+ * @brief Invalid packet identifier.
+ *
```

Comment:
I apologize, I checked in a file with an extra space left in mqtt.h, could you rebase uncrustify this file?

Reply:
Sure will do.